src/pages/quiz/Quiz.css

```diff
@@ -0,0 +1,50 @@
+.quiz-grid-layout {
+    display: grid;
+    grid-template-areas:
+      "... main ...";
+    gap: 2rem;
+  }
+  
+  .main-quiz,
+  .quiz {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+  
+  .main-quiz {
+    grid-area: main;
+    margin-top: 2rem;
+  }
+  
+  .quiz {
+    gap: 1rem;
+    border: 1px solid var(--COLOR-SECONDARY);
+    padding: 2rem;
+    border-radius: var(--BORDER-RADIUS); 
```

Comment:
appreciation - nice that you've a variable for border radius. Using this will make the UI consistent.